src/main/resources/schema/ans/0.5.9/utils/reference.json

```diff
@@ -44,7 +44,8 @@
         },
         "provider": {
           "description": "A URL that can resolve the id into an ANS element",
-          "type": "string"
+          "type": "string",
+          "format": "uri"
```

Comment:
I know this seems wrong, but I suspect we may have to leave `provider` alone, if only because a *lot* of code out there stubs in the empty string for a provider (owing to its being a required field, also wrong)

Possibly we could replace this with:


```
 "type": "string",
 "oneOf": [
   { 
     "enum": [ "" ]
    },
    {
      "format": "uri"
    }
  ]
```

Reply:
I agree it's better to be strict from the start.

We can't drop the field altogether, as it's needed for oembed objects.  The best we can do, I think, is make it non-required for all other reference types and force it into the restrictions above